src/components/InfoSection.js

```diff
@@ -19,4 +21,15 @@ const InfoSection = ({ headline, sectionId, children }) => {
   );
 };
 
+InfoSection.propTypes = {
+  headline: PropTypes.string.isRequired,
+  sectionId: PropTypes.string,
+  children: PropTypes.element,
+};
+
+InfoSection.defaultProps = {
+  sectionId: '',
```

Comment:
The problem with setting `sectionId` to an empty string by default is that the logic at the top of the component `const headingId = sectionId || headline.replace(/\s/g, '-').toLowerCase()` doesn't make that much sense anymore. `headingId` will always be `sectionId`. Only if you set `sectionId` to `null` headline will be used. I guess that's not what you wanted to achieve, right?